integration/integration_test.go

```diff
@@ -5040,11 +5040,12 @@ func testBPFSessionDifferentiation(t *testing.T, suite *integrationTestSuite) {
 
 	// Wait 10 seconds for both events to arrive, otherwise timeout.
 	timeout := time.After(10 * time.Second)
+outer:
```

Comment:
nit: Label should be capitalized

You can see an example here: https://go.dev/doc/effective_go#switch.